### ISSUE TITLE
Doesn't render anything on first pass.

### ISSUE THREAD

**Issue by damiangreen** [pvoznyuk/react-live-clock#18]
I'm using this component inside of a [ReactFitText ](https://github.com/gianu/react-fittext)component and as it doesnt appear to render anything on the first pass, the text isn't fitted (resized). Here's a section of my code with the workaround:

```
  public componentDidMount() {
    this.forceUpdate(); // Hack because text doesn't appear fitted on first render.
  }

  public render() {
    return (
      <Widget options={this.props.options}>
        <div role="presentation" className={s.root}>
          <ReactFitText compressor={0.3}>
            <h1 className={s.text}>
              <Clock format="HH:mm" ticking interval={1000} />
            </h1>
          </ReactFitText>
        </div>
      </Widget>
    );
  }
```

**Comment by pvoznyuk** [comment]
Hi @damiangreen it works fine for me, you can take a look at the example on this page: https://pvoznyuk.github.io/react-live-clock/

If it still doesn't work for you:
1/ try a new version (3.0.0) 
2/ try to add seconds to the time formatting to check if it works in that case

**Comment by anthony0030** [comment]
We have just published version 4.0.0 it is still in beta as I am still testing it.
Could you please check to see if you are having this problem with version:
"react-live-clock": "^4.0.0-beta"

It has a minimum react version "react": "^16.8" because it makes usage of react hooks.
